src/marks/Code.ts

```diff
@@ -30,7 +30,7 @@ export default class Code extends Mark {
 
   get schema() {
     return {
-      excludes: "strong em link mark strikethrough",
+      excludes: "_",
```

Comment:
Note: This is something of a coincidence – the `_` character in a prosemirror node spec means "any mark", it does not mean a literal underscore.